projects/Unit/APIApproval.Approve.verified.txt

```diff
@@ -90,6 +91,7 @@ namespace RabbitMQ.Client
         public System.Func<System.Collections.Generic.IEnumerable<RabbitMQ.Client.AmqpTcpEndpoint>, RabbitMQ.Client.IEndpointResolver> EndpointResolverFactory { get; set; }
         public System.TimeSpan HandshakeContinuationTimeout { get; set; }
         public string HostName { get; set; }
+        public System.Buffers.ArrayPool<byte> MemoryPool { get; }
```

Comment:
Give this goes into a minor do we really want to change the interface? I'm not against it I just want to mention we wanted to stay in the past for similar changes as much as possible semver compliant

Reply:
The change is just to `ConnectionFactory` and not the interface. I reviewed [this document](https://docs.microsoft.com/en-us/dotnet/core/compatibility/) and it seems this change falls into this category:

❓ REQUIRES JUDGMENT: Adding a new instance field to a type
This change impacts serialization

Reply:
Yeah you are right. Sorry I missed that it was on the class and not on the interface

Reply:
No worries. TBH I've always been a bit mystified by what seems to be interface-overuse in this library. Something to evaluate in 7.0.